src/solver/conv/conv_ocl_dir2D_bwdWrW_2.cpp

```diff
@@ -461,6 +461,17 @@ template <int N_BATCH_LOOPS>
 bool ConvOclBwdWrW2<N_BATCH_LOOPS>::IsApplicableBase(const ExecutionContext& ctx,
                                                      const ProblemDescription& problem) const
 {
+
+#if WORKAROUND_SWDEV_503936
+    // Disable this already deprecated solver for gfx11 and gfx12 because it works unstable for
+    // newer video cards
+    const auto device = ctx.GetStream().GetTargetProperties().Name();
+    if(miopen::StartsWith(device, "gfx11") || miopen::StartsWith(device, "gfx12"))
+    {
+        return false;
+    }
```

Comment:
```suggestion
    {
        const auto device = ctx.GetStream().GetTargetProperties().Name();
        if(miopen::StartsWith(device, "gfx11") || miopen::StartsWith(device, "gfx12"))
        {
            if(!env::enabled(MIOPEN_DEBUG_CONV_DIRECT_OCL_WRW2))
                return false;
        }
    }
```

Reply:
In case if you have env variable MIOPEN_DEBUG_CONV_DIRECT_OCL_WRW2 enabled, and you run on navi4 the library may crash. Crash should not happen on any input data and env variables setup. So, it looks safer to return false immediately without checking env variable.